### PR TITLE
Swagger: Show k8s APIs

### PR DESCRIPTION
**Pull request by ryantxu**
This updates the swagger UI so that a single page shows:
* shows v2 docs
* shows v3 docs
* includes all k8s docs (if available)
* works when running under a different app-root
* uses grafana logo

To see k8s endpoints, use:
```
[feature_toggles]
grafanaAPIServer = true
grafanaAPIServerWithExperimentalAPIs = true
```

### CODE REVIEW

File: public/views/swagger.html [hunk 1]
```diff
@@ -1,30 +1,34 @@
 <!-- HTML for static distribution bundle build -->
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
```
Comment (ryantxu) [inline public/views/swagger.html:5]: most of the changes are from running `format` -- but I think we want that, so i kept the changes.

File: public/views/swagger.html [hunk 2]
```diff
@@ -33,31 +37,56 @@
...
+          for (const [key, value] of Object.entries(apis.paths)) {
+            const parts = key.split('/');
+            if (parts.length == 3) {
+              urls.push({name: `${parts[1]}/${parts[2]}`, url: value.serverRelativeURL});
+            }
+          }
+        }
+        catch (err) {
+          console.warn('Error loading k8s apis');
+        }
```
Comment (ryantxu) [inline public/views/swagger.html:73]: ^^ lines 58 to 75 are the real change.  This adds each k8s service to the dropdown

**Review by charandas — APPROVED** [review]

File: public/views/swagger.html [hunk 3]
```diff
@@ -1,63 +1,96 @@
...
-        tagsSorter: "alpha",
-        tryItOutEnabled: true
-      });
-      // End Swagger UI call region
+      window.onload = async function () {
+        const urls = [
+            {name: 'Grafana API', url: "/public/api-merged.json"},
+        ];
+        try {
+          const rsp = await fetch('/openapi/v3');
```
[the remark this answers is not in the document]
Comment (ryantxu) [reply public/views/swagger.html]: Interesting!  i did not realize we had two copies of the UI.  I'll update both for now, and then explore some other option for making this more clear in the UI

**Comment by ryantxu** [comment]
@papagian -- I updated this so we now only have one swagger page and can pick the various API versions/flavors.  Can you take another look?

**Review by papagian — APPROVED** [review]

**Pull request merged**